### DISCUSSION TITLE
Re-escaping on recent directory completions

### DISCUSSION THREAD

**Discussion by tshu-w** [marlonrichert/zsh-autocomplete#314] [General]
<img width="1068" alt="Screen Shot 2021-07-30 at 14 45 06" src="https://user-images.githubusercontent.com/13161779/127612258-2e51b21c-82dd-4b04-afe5-07a91463727b.png">

when I type tab:
<img width="1101" alt="Screen Shot 2021-07-30 at 14 45 16" src="https://user-images.githubusercontent.com/13161779/127612269-84187d52-2594-48ed-99b1-15d31b8d5c82.png">

zsh 5.8 (x86_64-apple-darwin20.0)
This issue happens with cdr.

**Comment by marlonrichert** [comment]
Converted to issue #315.
